en/docs/guides/user-mgt/tenant-wise-config.md

```diff
@@ -0,0 +1,132 @@
+# Tenant-Wise Email Sender Configuration
```

Comment:
shall we start this page title with a verb? maybe configure tenant-wise email sender configuration